javascript/ql/lib/semmle/javascript/frameworks/NoSQL.qll

```diff
@@ -537,14 +542,22 @@ private module Mongoose {
       // NB: the complete information is not easily accessible for deeply chained calls
       f.getQueryArgument().getARhs() = result
     }
+
+    override DataFlow::Node getAResult() {
+      result = this.getCallback(this.getNumArgument() - 1).getParameter(1)
+    }
   }
 
-  class ExplicitQueryEvaluation extends DatabaseAccess {
+class ExplicitQueryEvaluation extends DatabaseAccess {
```

Comment:
```suggestion
  class ExplicitQueryEvaluation extends DatabaseAccess {
```
Looks like a spurious whitespace change.  
I also think the autoformatter will fail on this.